### PR TITLE
[onert] Support block quantization operand size calculation

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit update total_size() method to calculate block quantization type operand size.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #13742
Draft: #13693

### CODE REVIEW

File: runtime/onert/core/src/ir/OperandInfo.cc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ir/OperandInfo.h"
+
+#include <cassert>
```
Comment (glistening) [inline runtime/onert/core/src/ir/OperandInfo.cc:19]: (optional) Is this for future? This file seems not have `assert`.

File: runtime/onert/core/src/ir/OperandInfo.cc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+size_t OperandInfo::total_size() const
+{
+  const auto data_type = _typeInfo.type();
+  try
+  {
+    return _shape.num_elements() * sizeOfDataType(data_type);
+  }
+  catch (const std::runtime_error &e)
+  {
+    // Caclulate total size for ggml block quantization type on exception handling
```
Comment (zetwhite) [inline runtime/onert/core/src/ir/OperandInfo.cc:33]:
```suggestion
    // Calculate total size for ggml block quantization type on exception handling
```

**Review by zetwhite — DISMISSED** [review]
Without typo, the code looks good :+1:

**Review by zetwhite — APPROVED** [review]

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**